packages/icons/src/types/custom.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.svg' {
```

Comment:
This file is needed so typescript is able to recognized the `.svg` files and we can import them